tests/test_util/typing_test_data.py

```diff
@@ -121,7 +121,7 @@ def f25(a, b, /):
     pass
 
 
-def f26(x: Literal[1, 2, 3] = 1, y: Union[Literal['a'], Literal['b']] = 'a') -> None:
+def f26(x: Literal[1, 2, 3] = 1, y: Literal['a', 'b'] = 'a') -> None:
```

Comment:
Typing test data should continue to test unions of literals

Reply:
👍 Fixed.